prov/sockets/src/sock_wait.c

```diff
@@ -50,14 +50,17 @@ enum {
 
 int sock_wait_get_obj(struct fid_wait *fid, void *arg)
 {
+#ifndef _WIN32 /* there is no support of wait objects on windows */
 	struct fi_mutex_cond mut_cond;
+#endif /* _WIN32 */
 	struct sock_wait *wait;
 
 	wait = container_of(fid, struct sock_wait, wait_fid.fid);
 	if (sock_dom_check_manual_progress(wait->fab))
 		return -FI_ENOSYS;
 
 	switch (wait->type) {
+#ifndef _WIN32
 	case FI_WAIT_FD:
```

Comment:
unfortunately not. WIN32 API has calls to operate by socket descriptors to lock/wait, but there is no calls to wait for posix file descriptor (int fd). this is strong limitation and we decided to hide this API for windows platform (at least till we find appropriate replacement)